dvc/command/remove.py

```diff
@@ -42,7 +42,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    REMOVE_HELP = "Remove DVC-file outputs."
+    REMOVE_HELP = "Remove data files or directories tracked by DVC."
```

Comment:
This is not only about data, but about stages/models/etc too.

Reply:
Hmmm OK changed to "Remove DVC-tracked files or directories." both here and in the cmd ref now. Better? The part about removing entire stage files is explained in the cmd ref desc.